etc/firewalld/services/tendrl-monitoring-integration.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>tendrl-monitoring-integration</short>
+  <description>Enable port for tendrl monitoring integration</description>
```

Comment:
Could we note that this is needed for for grafana alerts callback here?

Reply:
Sure, I will add that description too